{
  "0 debug pnpm:scope": {
    "selected": 1
  },
  "1 info pnpm": {
    "err": {
      "name": "Error",
      "message": "not found: insta",
      "code": "ENOENT",
      "stack": "Error: not found: insta\n    at getNotFoundError (C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:41595:51)\n    at Function.whichSync [as sync] (C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:41672:13)\n    at getCommandAbsolutePathSync (C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:43237:44)\n    at default_1 (C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:43246:32)\n    at C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:178115:39"
    }
  },
  "2 error pnpm": {
    "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
    "prefix": "C:\\Users\\Acer\\MyProjects\\Klyntar\\Apollo",
    "err": {
      "name": "pnpm",
      "message": "not found: insta",
      "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
      "stack": "Error: not found: insta\n    at getNotFoundError (C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:41595:51)\n    at Function.whichSync [as sync] (C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:41672:13)\n    at getCommandAbsolutePathSync (C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:43237:44)\n    at default_1 (C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:43246:32)\n    at C:\\Users\\Acer\\AppData\\Roaming\\nvm\\v17.0.0\\node_modules\\pnpm\\dist\\pnpm.cjs:178115:39"
    }
  }
}